cli/src/commands/import/csv.ts

```diff
@@ -69,100 +67,195 @@ export default class ImportCSV extends BaseCommand<typeof ImportCSV> {
     const { file } = args;
     const {
       table,
```

Comment:
I think now it's the right time to remove `table` from being required.

```diff
table: Flags.string({
      description: 'The table where the CSV file will be imported to',
-     required: true
    }),
```

And we use it if provided, but we prompt to choose one or create one if not.

This has always bugged me, WDYT?

Reply:
This definitely makes sense and would be a good feature! 

My goal here is only to reach parity with the old implementation though. Any chance we can put it in list of future enhancements we could make list?!